lib/internal/source_map/source_map.js

```diff
@@ -169,28 +169,28 @@ class SourceMap {
   };
 
   /**
-   * @param {number} lineNumber in compiled resource
-   * @param {number} columnNumber in compiled resource
-   * @return {?Array}
+   * @param {number} 0-indexed line offset in compiled resource
+   * @param {number} 0-indexed column offset in compiled resource
+   * @return {object} representing start of range if found, or empty object
```

Comment:
The `object` type doesn't say much so I think it's better to declare an object with the properties that will/might be returned.

For example:
```js
/**
 * @returns {{
 *   foo: string;
 *   bar?: number;
 * }}
 */
```